packages/typespec-vscode/src/extension.ts

```diff
@@ -276,3 +289,29 @@ function showStartUpMessages(stateManager: ExtensionStateManager) {
     stateManager.cleanUpStartUpMessage(workspaceFolder.uri.fsPath);
   });
 }
+
+async function shendEmitterTelemetryData(activityId: string) {
```

Comment:
you can move this to typespec-utils.ts which should also be reused by the telemetry in emit-code